src/libraries/System.Runtime.Intrinsics/README.md

```diff
@@ -0,0 +1,23 @@
+# System.Runtime.Intrinsics
+Contains types used to create and convey register states in various sizes and formats for use with instruction-set extensions. Also exposes select instruction-set extensions for various architectures (`x86`, `Arm`, `Wasm`).
+
+Documentation can be found here: https://learn.microsoft.com/en-us/dotnet/api/system.runtime.intrinsics.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-System.Runtime.Intrinsics+label%3A%22help+wanted%22+) issues.
+
+## Source
+* `Vector64/128/256/512`: [../../coreclr/System.Private.CoreLib/src/System/Runtime/Intrinsics](../../coreclr/System.Private.CoreLib/src/System/Runtime/Intrinsics)
+* `Arm` intrinsics: [../../coreclr/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm](../../coreclr/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm)
```

Comment:
Some of these directories do not even exist.

Vast majority of the implementation is under src\libraries\System.Private.CoreLib\src\System\Runtime\Intrinsics.

There is a single x86 intrinsic method implemented in runtime-specific way, and that implementation is under all 

`src\coreclr\System.Private.CoreLib\src\System\Runtime\Intrinsics\X86`, `src\coreclr\nativeaot\System.Private.CoreLib\src\System\Runtime\Intrinsics\X86` and `src\mono\System.Private.CoreLib\src\System\Runtime\Intrinsics\X86`. In general, if you have an implementation path that points under coreclr, there should be also paths that point under nativeaot and mono.